eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -187,11 +187,16 @@ private static void run(final List<String> opts) throws Exception {
             phi.attr("Δ").put(new Data.Value<>(arg));
             app.attr(0).put(phi);
         }
-        if (!new Dataized(app).take(Boolean.class)) {
-            throw new IllegalStateException(
-                "Runtime dataization failure"
-            );
-        }
+        final Object result = new Dataized(app).take();
+        final String[] packages = app.toString().split("\\.");
+        Main.LOGGER.info(
+            String.format(
+                "The result of %s dataization is %s. Its value is %s",
+                packages[packages.length - 1],
```

Comment:
@yasamprom what if `app` equals to `foo`? It seems, there will be a runtime error here